launcher/Application.cpp

```diff
@@ -439,6 +430,15 @@ Application::Application(int &argc, char **argv) : QApplication(argc, argv)
         qDebug() << "<> Log initialized.";
     }
 
+    {
+        bool migrated = false;
+
+        if (!migrated)
+            migrated = handleDataMigration(dataPath, FS::PathCombine(QStandardPaths::writableLocation(QStandardPaths::AppDataLocation), "../../PolyMC"), "PolyMC", "polymc.cfg");
+        if (!migrated)
+            migrated = handleDataMigration(dataPath, FS::PathCombine(QStandardPaths::writableLocation(QStandardPaths::AppDataLocation), "../../multimc"), "MultiMC", "multimc.cfg");
```

Comment:
This structure seems a bit wierd, you can probably remove the ifs

Reply:
I could remove the first if, but the idea here was that there should only be one migration

Reply:
yeah i was mostly talking about the first one

Reply:
IMO, it makes the code look nicer and will be optimized out anyway